GoogleDataTransportCCTSupport/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v3.2.0
+- Expose a new public API to collect network or event code data.
```

Comment:
What are the clients of the new public API?  I'm worried about adding new API surface before sorting out the circular dependencies between GDT and GDTCCT.

Reply:
An internal team needs this functionality and implemented this feature on their own in #5767. Also, the deps aren't circular. Adopters need GDTCCT and GDTCCT needs GDT. Adopters -> GDTCCT -> GDT

Reply:
But the public API surface is in GDT so the header dependency tree doesn't match the linker dependency tree.  I'll get back to Swift PM in the next week or so and flush this out in more detail and hopefully find what I'm missing.